lib/loga/service_loggers/sidekiq_job_logger.rb

```diff
@@ -0,0 +1,43 @@
+module Loga
+  module ServiceLoggers
+    class SidekiqJobLogger
+      include LoggerUtilities
+
+      attr_reader :start
+
+      def initialize
+        @start = Time.now
+      end
+
+      def call(item, _queue)
+        event = ->(message) { Event.new(data: item, message: message, type: 'sidekiq') }
+
+        Sidekiq::Logging.with_context("source=#{item['class']}") do
+          begin
+            logger.info(event.call('start'))
```

Comment:
🙋‍♂️ I would like to see a similar approach taken in the `[Loga::Rack::Logger](https://github.com/FundingCircle/loga/blob/4f815e2cfdb3b978c06cb87b94264573cdcc84dc/lib/loga/rack/logger.rb#L25-L63)` where we collect data during the method execution and only create one log entry when it completes.

Admittedly the `Loga::Rack::Logger` could benefit from an initial log entry to indicate the request started. I think we could do this here at a debug level.